src/guide/working-with-pull-requests.md

```diff
@@ -24,7 +24,7 @@ Creating the PR is essentially the same as asking a large group of people to sta
 
 * Is the copyright statement at the top of each modified source file correct?
 
-* Did you run all relevant tests on the final version of the change? (Yes, I mean final! If you only knew how many times people _only changed a comment_ and caused a build failure.)
+* Did you run all relevant tests on the final version of the change? **Yes, I mean final!** If you knew how many times people _only changed a comment_ and caused a build failure you wouldn't question this.
```

Comment:
I agree with the sentiment here, but the tone is a little odd. It's one of only a couple places in the Guide where it refers to itself in the first person. When I run across wording like this I immediately think, "Who is the author?" In writing such as a blog post or newspaper column, there is a by-line, and so it's clear who is speaking. I don't think that's the case here, so it's just kind of strange.

The point is worth making though, so I'd suggest substituting something like,

> Did you run a full build and all relevant tests on the **final** version of the change? (It's important to do this on the truly final version, as even an apparently trivial change in a comment can break the build.)

The other use of first person is in testing-the-jdk.md; I'll make a comment there.